src/authentication/views.py

```diff
@@ -345,6 +428,10 @@ def post(self, request):
 
 
 class SudoView(APIView):
+    """
+    Promote a user to sudo
```

Comment:
That's not what this view does, when an admin hits this view, it will make their token impersonate a given user.